plugins/cryptolib/cryptolib.cpp

```diff
@@ -437,7 +437,7 @@ class CDSMCache
         else
         {
             IDigitalSignatureManager * pDSM = nullptr;
-            if (!isEmptyString(pubKeyFS) && !isEmptyString(privKeyFS))
+            if (!isEmptyString(pubKeyFS) || !isEmptyString(privKeyFS))
```

Comment:
@RussWhitehead  A reasonable change, but why does this fix the issue that was reported?  In that case they were both provided.